resources/campaigns/models/campaign.yml

```diff
@@ -30,6 +31,12 @@ allOf:
         description: An array of creatives that have been associated with this campaign.
         type: array
 
+      uploads:
+        description: A single-element array containing the upload object that is assocated with this campaign.
+        type: array
+        minItems: 0
+        maxItems: 1
```

Comment:
Just want to make sure to call this out, I assume this will be bumped in the future. So we are using an array of one to keep this flexible?

Reply:
@BennyKitchell in the future, if we allow multiple uploads/audience files per campaign, then yes, we'll remove the max. Right now, I decided to return the most recently created upload as it'll be the most relevant. Creatives is also a list of objects, but the same logic applies - the most recently created is the relevant one, and I might update creatives to match uploads to make things less confusing to users.